src/server-code/admin-server.js

```diff
@@ -219,6 +221,36 @@ export default function runAdminServer(serverFalcorModel) {
     });
   }
 
+  const CLIENT_ID = '870681894101-ce7tdp6h9fvt2jfrqaaalim1s2n48ie3.apps.googleusercontent.com';
```

Comment:
Keeping to the DRY principle we should not repeat anything twice, let's modularize this out, you can start a new `src/lib/constants.js` file if you want, or just put it in utilities as well. (the other place is in the Google Login react component)